layouts/partials/home/info.html

```diff
@@ -1,14 +1,17 @@
-{{ $why      := site.Params.why | markdownify }}
 {{ $features := site.Params.features }}
 <section class="section has-extra-padding">
   <div class="container has-text-centered">
     <p class="title is-size-1 is-size-2-mobile has-text-primary has-text-weight-medium">
       Why gRPC?
     </p>
-
-    <div class="content is-large has-text-weight-light">
-      {{ $why }}
-    </div>
+    <p class="content is-large has-text-weight-light">
```

Comment:
Putting the text directly in this partial rather than keeping it in the site config means anyone who wants to update the text needs to first track down where it lives. I don't see this as a win.

Reply:
Generally speaking, IMHO, config files are for config parameters, not content. IDE search works just fine to find any prose that might need to be updated.